src/tests/testing/model/GOTestWindchest.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2023-2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+
+#include <iostream>
+
+#include "GOTest.h"
+#include "GOTestCollection.h"
+#include "GOTestException.h"
+#include "GOTestWindchest.h"
+
+#include "GOOrganController.h"
+#include "config/GOConfig.h"
+#include "model/GOEnclosure.h"
+#include "model/GORank.h"
+#include "model/GOSoundingPipe.h"
+#include "model/GOWindchest.h"
+
+GOTestWindchest::~GOTestWindchest() {}
+
+void GOTestWindchest::run() {
+  wxString message;
+  // Check the size of Windchest is correct
+  unsigned w_size;
+  w_size = this->controller->AddWindchest(new GOWindchest(*this->controller));
+  message = "Windchest size is wrong";
```

Comment:
Ditto comments above.

Reply:
But you haven't change the code. Lars expected that you would write

``message = wxT("Windchest size is wrong")``

Reply:
The same pattern in all other places. You should use the `wxT` macro instead of assignment char* to wxSrtring

Reply:
Mmmh, ok.

This code was written at the beginning of this. IMHO, we shouldn't depends on wx in tests too apart for things related to dialogs. I think changing strings representations to std::string is the good approach.

Reply:
@oleg68 @larspalo I've changed the manner result message is defined.